server/database/migrations/2020_12_15_120531_add_like_column_to_favorites.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddLikeColumnToFavorites extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('favorites', function (Blueprint $table) {
+            $table->boolean('liked')->default(false);
+
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('favorites', function (Blueprint $table) {
+            if (Schema::hasColumn('favorites', 'liked'))
+            {
+                Schema::table('favorites', function (Blueprint $table)
+                {
+                    $table->dropColumn('liked');
+                });
+            }
+        });
```

Comment:
Focus you duplicate this line `Schema::table('favorites', function (Blueprint $table)`